### PR TITLE
Mano nft pages

### PR DESCRIPTION
**Pull request by xmano**
Add a landing page along with routing

### CODE REVIEW

File: apps/marketplace/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+
+
+@NgModule({
+  declarations: [],
+  imports: [
+    CommonModule
```
Comment (fritzschoff) [inline apps/marketplace/src/app/app-routing.module.ts]: here you would add the `RouterModule.forRoot([{path: '', component: AppComponent}])` and so on. If you want to have lazy loading, dont use the component key, use `loadChildren`

Comment (fritzschoff) [reply apps/marketplace/src/app/app-routing.module.ts]: or in this case your LandingComponent.

**Comment by xmano** [comment]
Added the route as mentioned but getting a run time issue:

image

File: apps/marketplace/src/app/app-routing.module.ts (hunk 1 above)
Comment (xmano) [reply apps/marketplace/src/app/app-routing.module.ts]: Fixed the issue by defining the correct module.. used the right CLI.

File: apps/marketplace/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+const routes: Routes = [
+  {
+    path: '',
+    redirectTo: 'landing',
+    pathMatch: 'full'
+  },
+  {
+    path: 'landing',
+    loadChildren: () => import('./landing/landing.module').then(m => m.LandingModule)
```
Comment (fritzschoff) [inline apps/marketplace/src/app/app-routing.module.ts:12]: can't find the LandingModule file in your PR?

**Review by fritzschoff — APPROVED** [review]

File: apps/marketplace/src/app/app-routing.module.ts (hunk 2 above)
Comment (xmano) [reply apps/marketplace/src/app/app-routing.module.ts:12]: good catch!.. thanks :)

Comment (xmano) [reply apps/marketplace/src/app/app-routing.module.ts:12]: git added the files.

**Pull request merged**